indico/MaKaC/webinterface/pages/abstracts.py

```diff
@@ -914,7 +914,7 @@ def _getTrackItemsHTML( self ):
 
     def _getSessionItemsHTML( self ):
         items = [ i18nformat("""<option value="conf">--_("no session")--</option>""")]
-        for session in self._conf.getSessionList():
+        for session in sorted(self._conf.getSessionList(), key=lambda s: s.getTitle()):
```

Comment:
Not really important here, but you could even use `key=methodcaller('getTitle')` (with `from operator import methodcaller`)